apps/backend/src/models/User.ts

```diff
@@ -23,6 +29,13 @@ class User extends ParanoidModel {
 
   @BeforeCreate
   static async makeUpperCase(instance: User) {
+    console.log(instance)
+    instance.password = await argon2.hash(instance.password)
+  }
+
+  // @BeforeBulkUpdate
+  @BeforeUpdate
+  static async hashUpdatedPassword(instance: User) {
```

Comment:
I think it can be merged into one function. 

```ts
@BeforeCreate
@BeforeUpdate
static async hashPassword(instance: User) { }
```

Also can you change the function name from `makeUpperCase` to `hashPassword`